.github/workflows/sast-linters.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     runs-on: ubuntu-22.04
     env:
       GO111MODULE: on
+      GOTOOLCHAIN: auto
```

Comment:
This fixes an error we have with the CI https://github.com/zeta-chain/node/actions/runs/13109842437/job/36571179228
I tested patching the action https://github.com/lumtis/gosec/commit/fad752daec298a16ceda9b439850430c5a30ab31 but it didn't solve the issue